src/app/app.component.html

```diff
@@ -358,6 +358,15 @@
                             </li>
                           </ul>
 
+                          <!-- Response duplication button -->
+                          <button
+                            *ngIf="!(deleteCurrentRouteResponseRequested$ | async)"
+                            type="button" id="duplication-button" class="btn btn-link btn-icon"
```

Comment:
Could you please use a more specific id like `route-response-duplication`, just in case. 
Also, if you have time could you please add an ID on the deletion button too, to use it in the tests? I realize that the selector is quite cumbersome (`.this:not(.that)...`)